backend/src/main/resources/graphql/main.graphqls

```diff
@@ -50,6 +50,7 @@ type DeviceType {
   model: String
   loincCode: String
   swabType: String
+  testLength: Int
```

Comment:
Just to clarify - I assume this means the updateFacility and addFacility mutations, right? Since they're currently accepting a string array and they'll need to accept the testLength int as well. Or are there other mutations you're referring to?

Reply:
Those are the mutations I was referring to, yes. 🙂

Reply:
Gotcha, thank you :) I'm going to implement those changes in a followup PR, since this one is ostensibly just a timer change. The issue for that is [here](https://github.com/CDCgov/prime-simplereport/issues/1348)